build.xml

```diff
@@ -78,6 +78,7 @@
     <property name="test.simulator-test.src" value="${test.dir}/simulator/test"/>
     <property name="test.driver.connection_timeout_ms" value="5000"/>
     <property name="test.driver.read_timeout_ms" value="12000"/>
+    <property name="test.jvm.args" value="" />
```

Comment:
this is to allow setting test jvm args from `ant`. MCK didn't like the idea of using environment variables to set this, and ant doesn't propagate by default, so this lets us to define in CI and have ant pass to the test JVM

Reply:
Could we use a different name here to reflect that? This gives us "test.jvm.args" and "test-jvmargs" which is pretty confusing. Maybe something like "ant.jvm.args" or something?

Reply:
all these settings are for `ant`, so I rather not use `ant.jvm.args` as that I feel makes things more confusing (ant is a java project, so these sound to change the JVM args of ant itself).

`test-jvmargs` is an implementation detail where its trying to have different defaults for j8 and j11; the names are similar true, but I rather rename `test-jvmargs` than `test.jvm.args` as this one is meant to be used from CI

Reply:
went with `_std-test-jvmargs`